src/utils/clmfates_interfaceMod.F90

```diff
@@ -481,7 +482,7 @@ subroutine init(this, bounds_proc )
          call this%init_soil_depths(nc)
          
          if (use_fates_planthydro) then
-            call InitHydrSites(this%fates(nc)%sites,this%fates(nc)%bc_in,numpft_fates)
```

Comment:
Looking at the change in the fates PR it looks like `numpft` might still be needed for `InitHydrSites`?

Reply:
looks consistent to me: https://github.com/NGEET/fates/blob/320a8fc0875e228df4133544f122c83844d8720e/biogeophys/FatesPlantHydraulicsMod.F90#L1242